lib/postgresql/batch-client.js

```diff
@@ -50,16 +50,45 @@ function runQueries(queryRunner, queries, callback) {
     }
 
     async.eachSeries(
-        queries,
+        queries.query,
         function(query, done) {
-            queryRunner.run(query, false, done);
+            if (typeof query === 'string') {
```

Comment:
Definitely this has to be improved. Because fallback-jobs, we need go through it and run queries with or without their corresponding fallback.